modules/user-guide/pages/gitlab-code-quality-reports.adoc

```diff
@@ -0,0 +1,16 @@
+// Metadata for Antora
+:navtitle: GitLab Code Quality reports
```

Comment:
Please add `GitLab Code Quality` in `.vale/styles/RedHat/Headings.yml` exceptions (line 71):

```
  - GitLab Code Quality
```

And a test in `.vale/fixtures/RedHat/Headings/testvalid.adoc`

```

== GitLab Code Quality

```